config.yaml

```diff
@@ -7,12 +7,16 @@ options:
   # An example config option to customise the log level of the workload
   log-level:
     description: |
-      Configures the log level of gunicorn.
+      Configures the log level of gunicorn. 
```

Comment:
People using this software don't know or care that it's gunicorn in the background, you may want to be more general